tests/test_scheduler_workstacean.py

```
@@ -0,0 +1,168 @@
+        adapter.add_job("the actual prompt", "0 9 * * *", job_id="x")
+        inner = recorder.calls[0]["json"]["payload"]["payload"]
+        assert inner["content"] == "the actual prompt"
+        assert inner["channel"] == "a2a"
+        assert inner["agent_name"] == "gina-personal"
+
+    def test_iso_oneshot_accepted(self, adapter, recorder):
+        adapter.add_job("hi", "2099-01-01T00:00:00", job_id="x")
+        assert len(recorder.calls) == 1
+
+    def test_malformed_schedule_rejected(self, adapter):
+        with pytest.raises(ValueError):
```

Ruff (0.15.12)

[warning] `pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)